.github/workflows/cpp-python-tests.yml

```diff
@@ -7,102 +7,116 @@ on:
 
 jobs:
   build:
-
     runs-on: ubuntu-20.04
```

Comment:
Test on ubuntu-18.04 ?

Reply:
Added Python 3.9 and 3.10, and ubuntu-18.
Python 3.7 won't be tested as it is not in the matrix.